Description: info: direntry description and infodir category
 Set both description for directory entry and infodir category.
Author: Andreas Metzler <ametzler@debian.org>
Origin: vendor
Forwarded: http://bugs.exim.org/show_bug.cgi?id=925
Last-Update: 2013-09-29

--- a/doc-docbook/filter.xfpt
+++ b/doc-docbook/filter.xfpt
@@ -79,6 +79,12 @@
 <copyright><year>
 .copyyear
            </year><holder>University of Cambridge</holder></copyright>
+<abstract role="texinfo-node">
+  <para>Exim filtering</para>
+</abstract>
+<subjectset scheme="texinfo-directory">
+  <subject><subjectterm>Mail</subjectterm></subject>
+</subjectset>
 </bookinfo>
 .literal off
 
--- a/doc-docbook/spec.xfpt
+++ b/doc-docbook/spec.xfpt
@@ -185,6 +185,12 @@
 <copyright><year>
 .copyyear
            </year><holder>University of Cambridge</holder></copyright>
+<abstract role="texinfo-node">
+  <para>The Exim MTA</para>
+</abstract>
+<subjectset scheme="texinfo-directory">
+  <subject><subjectterm>Mail</subjectterm></subject>
+</subjectset>
 </bookinfo>
 .literal off
 
